README.md

```diff
@@ -33,6 +35,13 @@ functionality exist in the file `ray_beam_runner/portability/ray_runner_test.py`
 
 ```shell
 pytest ray_beam_runner/portability/ray_runner_test.py
+
+# If saw error messages like "TypeError: The type of keyword 'num_returns' must be (<class 'int'>, <class 'NoneType'>), but received type <class 'str'>"
+# pick the nightly release works for your platform: https://docs.ray.io/en/latest/ray-overview/installation.html#daily-releases-nightlies
+# try to install ray nightly locally, for example:
+pip install -U "ray @ https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp39-cp39-manylinux2014_x86_64.whl"
```

Comment:
Ray 2.1 will be released soon, after that we can change setup.py to require `ray >= 2.1` so we no longer need to install nightly.

Reply:
sounds good to me, currently I added it as an error fallback step. do you have a rough release date? probably I can have another pr by then

Reply:
Yea, end of this week or early next week.